<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Articles</class>
 <widget class="QWidget" name="Articles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 182, 168);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>521</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter article</string>
    </attribute>
    <widget class="QLabel" name="Narticle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom article</string>
     </property>
    </widget>
    <widget class="QLabel" name="Carticle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Categorie</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rarticle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rayon</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_na">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ca">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ra">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_retour">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pa">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>521</width>
       <height>391</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajouter article</string>
      </attribute>
      <widget class="QLabel" name="Narticle_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom article</string>
       </property>
      </widget>
      <widget class="QLabel" name="Carticle_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QLabel" name="Rarticle_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rayon</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_na_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ca_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ra_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_retour_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pa_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>220</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Afficher articles</string>
      </attribute>
      <widget class="QTableView" name="tabetudiant_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>331</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher articles</string>
    </attribute>
    <widget class="QTableView" name="tabetudiant">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>331</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
